### PR TITLE
CANS-402 Remove roles filter from the validation

### PR DESCRIPTION
**Pull request by nasar-cambria**
## Description
Removing Roles from the permission filter.

## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by nasar-cambria** [comment]
re-test this please

File: spec/lib/infrastructure/cwds_permission_checker_spec.rb [hunk 1]
```diff
@@ -24,35 +24,12 @@ module Infrastructure
       context 'when there is a valid session' do
         it 'user is able to access the application with valid roles and privileges' do
```
Comment (LeonidMarushevskyi) [inline spec/lib/infrastructure/cwds_permission_checker_spec.rb]: Probably need to remove "roles" from test description

Comment (edgibbs) [reply spec/lib/infrastructure/cwds_permission_checker_spec.rb]: :+1

Comment (nasar-cambria) [reply spec/lib/infrastructure/cwds_permission_checker_spec.rb]: removed.

**Pull request merged**